requirements/torch.txt

```diff
@@ -1,2 +1,3 @@
 torch>=1.8.0
 tensorboard>=2.4.0
+pytorch-lightning>=1.5.8
```

Comment:
If you haven't done so already, might be worth testing with older versions to see if it also works, in order to have the loosest possible requirements here.

Reply:
Yes, 1.5.0 is the lowest supported from what I've found.